oak-segment-tar/src/test/java/org/apache/jackrabbit/oak/segment/NodeRecordTest.java

```diff
@@ -105,7 +105,7 @@ public void baseNodeStateShouldBeReusedAcrossGenerations() throws Exception {
 
             SegmentWriter writer = defaultSegmentWriterBuilder("test")
                     .withGeneration(generation)
-                    .withWriterPool()
+                    .withWriterPool(SegmentBufferWriterPool.PoolType.GLOBAL)
```

Comment:
Given my other comment about keeping `withWriterPool()`, I suggest reverting this changes.